src/components/LineBetween/LineBetween.js

```diff
@@ -0,0 +1,206 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { isNil } from 'lodash';
+
+import { getElement } from './utils';
+import { Line } from './Line';
+
+const anchorTextValues = {
+  x: {
+    left: 0,
+    center: 0.5,
+    right: 1,
+  },
+  y: {
+    top: 0,
+    center: 0.5,
+    bottom: 1,
+  },
+};
+
+const zIndex = 20;
+
+export const LineBetween = props => {
+  const {
+    from,
+    to,
+    container,
+    fromAnchor,
+    toAnchor,
+    fromAnchorOffset = '',
+    toAnchorOffset = '',
+    orientation = 'vertical',
+  } = props;
+
+  const fromElement = getElement(from);
+  const toElement = getElement(to);
+
+  if (!fromElement || !toElement) {
+    return null;
+  }
+
+  const fromElementDimensions = fromElement.getBoundingClientRect();
+  const toElementDimensions = toElement.getBoundingClientRect();
+
+  const parseAnchorText = (value, axis) => anchorTextValues[axis][value];
+
+  const parsePixelValue = value => {
```

Comment:
I think `parsePixelValue`, `parseAnchorOffset`, `parseAnchorPercent`, `getAnchorValue` do not work with this at all and can be moved out of the component. I would move all that logic to some other file and just import it there. Maybe it makes sense to gather all this under some es6 `class`.